Godeps/_workspace/src/k8s.io/kubernetes/cmd/kube-apiserver/app/plugins.go

```diff
@@ -26,7 +26,7 @@ import (
 	// Admission policies
 	_ "k8s.io/kubernetes/plugin/pkg/admission/admit"
 	_ "k8s.io/kubernetes/plugin/pkg/admission/deny"
-	_ "k8s.io/kubernetes/plugin/pkg/admission/exec/denyprivileged"
```

Comment:
deprecated upstream in favor of `DenyEscalatingExec` which is covered by the line below (which also registers itself under the old plugin name)